### ISSUE TITLE
diffrent results in 0.10 and iojs

### ISSUE THREAD

**Issue by calvinmetcalf** [crypto-browserify/pbkdf2-compat#9]
password: test
salt (in hex format): 7b970da6a2fddaba
iterations: 1000
length: 64
algo: sha512

gives me 

```
e6d98ba1993f1e721dab924cec8ca8e7af4b40035553da88aec486131afc0f88ff40a29467911fb58c042ede9701551f0b4b06b4207b2a831d90783aab3095da
```

in 0.10

but in iojs

```
7c89cceaea1811b0aeaf23587fadb4b80580d9b26c7e7489ec56f9f0a84c1b612c788ddfb4a86dee8e5be70d4742b84ff3e818e3b83aa68d21db5b4d9f70dc8e
```

**Comment by calvinmetcalf** [comment]
ok figured out the issue, I think it has to do with utf encoding of salts

**Comment by dcousens** [comment]
Fixed by #10
